tests/khmer_tst_utils.py

```diff
@@ -53,7 +53,27 @@ def cleanup():
     cleanup_list = []
 
 
+def scriptpath(scriptname='interleave-reads.py'):
+    "Return the path to the scripts, in both dev and install situations."
+
+    # note - it doesn't matter what the scriptname is here, as long as
+    # it's some khmer script present in this version of khmer.
+
+    path = os.path.join(os.path.dirname(__file__), "../scripts")
+
+    if os.path.exists(os.path.join(path, scriptname)):
+        return path
+
+    for path in os.environ['PATH'].split(':'):
+        if os.path.exists(os.path.join(path, scriptname)):
```

Comment:
On Mon, Jul 27, 2015 at 09:51:47AM -0700, Michael R. Crusoe wrote:
> > @@ -53,7 +53,27 @@ def cleanup():
> >      cleanup_list = []
> >  
> >  
> > +def scriptpath(scriptname='interleave-reads.py'):
> > +    "Return the path to the scripts, in both dev and install situations."
> > +
> > +    # note - it doesn't matter what the scriptname is here, as long as
> > +    # it's some khmer script present in this version of khmer.
> > +
> > +    path = os.path.join(os.path.dirname(__file__), "../scripts")
> > +
> > +    if os.path.exists(os.path.join(path, scriptname)):
> > +        return path
> > +
> > +    for path in os.environ['PATH'].split(':'):
> > +        if os.path.exists(os.path.join(path, scriptname)):
> 
> For what circumstance do we need to search the system path? Seems dangerous.

This is a refactor of existing code (merged by you, I believe :), so we were
already doing this.  I couldn't think of a better way. Punt to an issue if you
want to discuss, but this isn't new functionality.